### DISCUSSION TITLE
Function hide profile name above your head

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#8279] [Feature Requests]
Good time of day. I have such a problem that I can't hide my account name above my head. I use nameplate prefixes and suffixes, and also hide my account name. But the name is still displayed above my head, no matter how hard I try. I bought your plugin and I love it very much, but I really need the function of hiding my account name through the cmi plugin. Please help me.

**Comment by kirmraft** [comment]
I have the same issue.
